### PR TITLE
3/14 daily (adj hood pog)

### PR DESCRIPTION
**Pull request by jude-lif**

### CODE REVIEW

File: src/main/java/frc/team3128/RobotContainer.java [hunk 1]
```diff
@@ -729,7 +729,8 @@ private void initLimelights(Limelight... limelightList) {
       
         for (Limelight ll : limelightList) {
             NarwhalDashboard.addLimelight(ll);
-            ll.turnLEDOff();
+            // ll.turnLEDOff();
+            ll.turnLEDOn();
```
Comment (danielwang546) [inline src/main/java/frc/team3128/RobotContainer.java]: Probably revert this to turn off?

File: src/main/java/frc/team3128/subsystems/Hood.java [hunk 2]
```diff
@@ -69,6 +69,7 @@ public void zeroEncoder() {
     }
 
     public void startPID(double angle) {
+        angle = ConstantsInt.ShooterConstants.SET_ANGLE;
```
Comment (danielwang546) [inline src/main/java/frc/team3128/subsystems/Hood.java]: Delete this line so as to use the value passed in, not the one from dashboard

File: src/main/java/frc/team3128/subsystems/Shooter.java [hunk 3]
```diff
@@ -128,6 +129,7 @@ public void startPID() {
      * Begins the PID loop to achieve the desired RPM with the currently set Shooter State
      */
     public void startPID(double rpm) {
+        rpm = ConstantsInt.ShooterConstants.SET_RPM;
```
Comment (danielwang546) [inline src/main/java/frc/team3128/subsystems/Shooter.java]: Should be removed to use the value passed in and not the one from dashboard.

File: src/main/java/frc/team3128/Constants.java [hunk 4]
```diff
@@ -157,7 +157,7 @@
 
         public static final int HOOD_MOTOR_ID = 5;
 
-        public static final double kP = 0.3128;//0.4;//0.3128;//0.254;
+        public static final double kP = 0.28;//0.4;//0.3128;//0.254;
```
Comment (danielwang546) [inline src/main/java/frc/team3128/Constants.java:160]: The true hood angles we observed were pretty close to 1 degree above setpoint every time - hood PID should be retuned, but not at the cost of the data we collected today

File: src/main/java/frc/team3128/subsystems/Shooter.java [hunk 5]
```diff
@@ -227,16 +228,7 @@ public void simulationPeriodic() {
...
-
-        return 0.00971 * Math.pow(dist, 3) - 0.289 * Math.pow(dist, 2) - 52.17 * dist + 5196 + 600;
-        // double rpm;
-        // if (dist < 78) {
-        //     rpm = 17.7 * dist + 2187;   
-        // } else {
-        //     rpm = -4.54 * Math.pow(dist, 2) + 917 * dist - 40328;
-        // }
-        // return rpm + 1000;
+        return -1.43648019e-3*dist*dist*dist + 4.42551199e-1*dist*dist - 3.02450570e1*dist + 3.16957933e3 - 100;
```
Comment (danielwang546) [inline src/main/java/frc/team3128/subsystems/Shooter.java:231]: Likely need to figure out if we need any offset for what the eqn. gives during driver practice tomorrow

**Review by danielwang546 — APPROVED** [review]

File: src/main/java/frc/team3128/RobotContainer.java (hunk 1 above)
Comment (jude-lif) [reply src/main/java/frc/team3128/RobotContainer.java]: Yes will do this right now

**Pull request merged**